src/gui/src/SongEditor/SongEditorPanel.cpp

```diff
@@ -950,3 +920,7 @@ void SongEditorPanel::timelineActivationEvent( int nEvent ){
 	
 	m_pPositionRuler->createBackground();
 }
+
+void SongEditorPanel::updateSongEditorEvent( int ) {
+	updateAll();
```

Comment:
This has an undesirable side-effect (`updateAll` -> `resyncExternalScrollbar`) of scrolling the Song Editor viewport to bring the currently selected pattern into view, which was added so that newly created patterns are visible.

When the user clicks to set a cell in the song editor, but the current pattern isn't visible, the Song Editor will move after the click to show the currently selected pattern.

I'm making some changes to song editor scrolling, so I'll fix it there.

Reply:
Ah, your right. Sorry for that!

Reply:
But, on the other hand, for setting/removing a pattern resyncing the scrollbar is important since this also changes the current pattern number. Probably calls for different arguments provided to the event.